aws/resource_aws_macie_s3_bucket_association_test.go

```diff
@@ -0,0 +1,174 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/macie"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSMacieS3BucketAssociation_basic(t *testing.T) {
```

Comment:
Is there a way to programatically enable/disable Macie?

```
=== RUN   TestAccAWSMacieS3BucketAssociation_basic
--- FAIL: TestAccAWSMacieS3BucketAssociation_basic (6.07s)
	testing.go:518: Step 0 error: Error applying: 1 error(s) occurred:
		
		* aws_macie_s3_bucket_association.test: 1 error(s) occurred:
		
		* aws_macie_s3_bucket_association.test: Error creating Macie S3 bucket association: InvalidInputException: The request was rejected. Macie is not enabled for this AWS account *******.
```

Reply:
I'll see if [`AssociateMemberAccount`](https://docs.aws.amazon.com/sdk-for-go/api/service/macie/#Macie.AssociateMemberAccount) with the current account ID does that.
I just manually enabled Macie via the console.

Reply:
No, it looks like Macie cannot programatically be enabled:

```console
$ aws --region us-east-1 macie list-member-accounts

An error occurred (InvalidInputException) when calling the ListMemberAccounts operation: The request was rejected. Macie is not enabled for this AWS account 999999999999.
$ aws --region us-east-1 macie associate-member-account --member-account-id 999999999999

An error occurred (InvalidInputException) when calling the AssociateMemberAccount operation: The request was rejected. Macie is not enabled for this AWS account 999999999999.
```

I'll add a link to the manual setup instructions to the documentation.